qml/StatisticsChartBox.qml

```diff
@@ -174,6 +175,13 @@ Rectangle {
                         statisticsChartBox.destroy()
                     }
                 }
+                Action {
+                    text: "Export all to CSV"
+                    onTriggered: {
```

Comment:
The application freezes when saving a csv while a dynamic chart is running.

```suggestion
                    enabled: (isDynamic && !running) || (!isDynamic)
                    onTriggered: {
```

Reply:
:+1: